main/acle.rst

```diff
@@ -1820,6 +1825,9 @@ using the A key for signing.
 The protection applied to any particular function may be overriden by
 mechanisms such as function attributes.
 
+``__ARM_FEATURE_PAUTH`` is defined to 1 if Pointer Authentication extension
+is available on the target. It is undefined otherwise.
```

Comment:
Same here - a bit of explanation on why you need a new macro would help (n a design document).